dashboard/components/DevDashboard.vue

```diff
@@ -21,6 +21,7 @@
 					class="campaign current-branch"
 					:campaign="currentCampaign"
 					:compile-info="compileInfo[ currentCampaign.banners.ctrl.pageName ]"
+					:pull-request-url="campaignUrlMap[currentCampaign.campaign] as string"
```

Comment:
```suggestion
					:pull-request-url="campaignUrlMap[currentCampaign.campaign]"
```

The type casting in the template is not necessary, you already defined the type of `campaignUrlMap` (see also line 34)

Reply:
Done.